home0.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import ("fmt")
+
+var a, x, y, p, sq, x1, y1 int

```

Comment:
We don't need to declare package-level variables here.
It's OK to move declaration to main func, or even use `:=` to avoid separated declaration